engine/wwtlib/WWTControl.cs

```diff
@@ -2549,6 +2563,18 @@ public Imageset GetImagesetByName(string name)
             return null;
         }
 
+        public Imageset GetImageSetByUrl(string url)
+        {
+            foreach (Imageset imageset in ImageSets)
+            {
+                if (imageset.Url.ToLowerCase() == url.ToLowerCase())
```

Comment:
Was there a specific reason to case-fold the URLs? I feel like it would be better to not do that unless there's a strong reason.

Reply:
The only reason I did it was to be more forgiving to users typos